wlauto/workloads/msword/__init__.py

```diff
@@ -79,12 +80,18 @@ class MsWord(AndroidUiAutoBenchmark):
                   The output is piped to log files which are then pulled from the phone.
                   '''),
         Parameter('use_test_file', kind=bool, default=False,
-                  description='If ``True`` then use a provided test file instead of creating one'),
-        Parameter('test_file', kind=str, description='Document to push to the device for testing'),
-        Parameter('test_image', kind=str, description='Image to be embedded in the document under test'),
```

Comment:
The reason for this was just to have it in-line with powerpoint workload, as they both do very similar things here.
I believe once we have the dependency repo up and running, this sort of thing will be removed in favour of using that resource_getter, so it will all change again.

Happy for now to keep this as a parameter though.